demos/python_demos/im2latex_demo/README.md

```diff
@@ -0,0 +1,105 @@
+# Im2LaTeX Python\* Demo
+
+This demo shows how to run im2latex models. Im2latex models allow to get a latex formula markup from the image.
+
+> **NOTE**: Only batch size of 1 is supported.
+
+## How It Works
+
+The demo application expects an im2latex model that is split into two parts. Every model part must be in the Intermediate Representation (IR) format.
+
+The First model is Encoder which extracts features from an image and prepares first steps of the decoder.
+
+* One input is `imgs` for input image 
```

Comment:
It could be very bad strategy to fix names of I/O, since it is changeable. The best practice is to set them via demo keys, or derive in the code basing on layer shape etc.

Reply:
Inputs and outputs names derivation basing on tensor shape doesn't work here. Some of the inputs have identical shapes.

it's not very good option to tell the demo I/O names via demo keys as we still will need to update key default values if I/O names change. But that's the approach we have. @morkovka1337, please add the keys.

Reply:
Added keys. Feel free to suggest a better way as I am not sure adding so many optional arguments is OK